src/CLI/actioner/sonic-cli-ifa.py

```diff
@@ -153,11 +149,22 @@ def get_tam_ifa_flow_stats(args):
                 api_response = response.content['sonic-ifa:TAM_INT_IFA_FLOW_TABLE']['TAM_INT_IFA_FLOW_TABLE_LIST']
             else:
                 api_response = response.content['sonic-ifa:TAM_INT_IFA_FLOW_TABLE_LIST']
+
+            print(api_response)
             for i in range(len(api_response)):
-                acl_counter_key = 'COUNTERS:' + api_response[i]['acl-table-name'] + ':' + api_response[i]['acl-rule-name']
-                flow_stats = counters_db.get_all(counters_db.COUNTERS_DB, acl_counter_key)
-                api_response[i]['Packets'] = flow_stats['Packets']
-                api_response[i]['Bytes'] = flow_stats['Bytes']
+                rule = api_response[i]['acl-rule-name']
+                print(rule)
+                # tokenize the rulename with '_' and fetch last number
+                tmpseq = (rule.split("_", 1))[-1]
+                print(tmpseq)
+                path = cc.Path('/restconf/data/openconfig-acl:acl/acl-sets/acl-set={name},{type}/acl-entries/acl-entry={seqid}', name=api_response[i]['acl-table-name'], type="ACL_IPV4", seqid=tmpseq)
+                flow_stats = api.get(path)
```

Comment:
We should not be hard-coding type...it can be ipv6 also

Reply:
@asha-behera I did thought about it. But a flow only contains acl table/rule and a collector. Should I read collector properties and decide if I have to use v4 or v6 with acl type?

Reply:
The type should be the type of ACL, I think IP_TYPE field has been added in OC ACL to know the type of ACL. I am not sure if the change has been merged.  I not then we can use ACLNAME which would have IPV4 or IPV6 appended to it like (..e.g TestACL100_ACL_IPV4)

Reply:
Using get_openconfig_acl_acl_acl_sets() to fetch acl list to index into the table/rule to get counters.